app/src/main/java/com/framgia/wsm/data/source/RequestRepository.java

```diff
@@ -140,7 +140,7 @@ public RequestRepository(RequestRemoteDataSource remoteDataSource) {
         requestOverTime2.setCreatedAt("23/09/2017");
         requestOverTime2.setFromTime("21/09/2017 18:00");
         requestOverTime2.setToTime("22/09/2017 19:00");
-        requestOverTime2.setStatus(1);
+        requestOverTime2.setStatus("pending");
```

Comment:
ERROR: [WrongConstant] Must be one of: StatusCode.ACCEPT_CODE, StatusCode.PENDING_CODE, StatusCode.REJECT_CODE
 Ensures that when parameter in a method only allows a specific set of constants, calls obey those rules.